downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -220,7 +220,7 @@ abstract class VersionControlSystem {
     /**
      * Return the Java class name as a simple way to refer to the [VersionControlSystem].
      */
-    override fun toString(): String = javaClass.simpleName
+    fun getType(): String = javaClass.simpleName
```

Comment:
This should be a property: `val type = javaClass.simpleName`

Reply:
Also, this commit misses at least one call site of `toString()`:

https://github.com/heremaps/oss-review-toolkit/blob/b596eda3a757307b3a30c8aaad43d7aed0d76538/downloader/src/main/kotlin/Downloader.kt#L96

Please use the IntelliJ's "Find usages" (Alt-F7) to check. I'm not sure whether that also find implicit calls to `toString()` inside of string messages, though.

Reply:
> This should be a property: val type = javaClass.simpleName

I basically agree, but I'd prefer to do that separately. We have a bunch of more getters in here that could / should be properties, but for this commit I'd prefer to keep the diff as small as possible and not change semantics where not required.